vmdb/spec/factories/orchestration_stack.rb

```diff
@@ -1,4 +1,28 @@
 FactoryGirl.define do
   factory :orchestration_stack do
   end
+
+  factory :orchestration_stack_openstack_infra, :class => "OrchestrationStackOpenstackInfra" do
+    after :create do |x|
+      x.parameters << FactoryGirl.create(:orchestration_stack_parameter_openstack_infra_compute)
+      x.parameters << FactoryGirl.create(:orchestration_stack_parameter_openstack_infra_controller)
+    end
+  end
+
+  factory :orchestration_stack_parameter_openstack_infra, :class => "OrchestrationStackParameter" do
+  end
+
+  factory :orchestration_stack_parameter_openstack_infra_compute, :parent => :orchestration_stack_parameter_openstack_infra do
+    after :create do |x|
+      x.name="compute-1::count"
```

Comment:
rubocop would not like the missing space around '=' here, if it was run on `spec`

the same below